Testscripts/Linux/TIME-CLOCKSOURCE.sh

```diff
@@ -117,25 +118,24 @@ function UnbindCurrentSource()
 #
 # MAIN SCRIPT
 #
-GetDistro
 case $DISTRO in
-	redhat_6 | centos_6 | redhat_7 | centos_7)
+	redhat_6 | centos_6 | redhat_7 | centos_7 | debian*)
 		LogMsg "WARNING: $DISTRO does not support unbind current clocksource, only check"
 		CheckSource
 		;;
 	redhat_8 |centos_8|fedora*|clear-linux-os)
 		CheckSource
 		UnbindCurrentSource
 		;;
-	ubuntu*|debian*)
+	ubuntu*)
```

Comment:
Can we combine rest of the cases with `| (or)` if we are doing same steps in each of them?

Reply:
Fixed in 86558f1016853b7f4530044bc67f5cc65ee4adb2